src/command_state.h

```diff
@@ -52,8 +52,9 @@ class CommandState {
   // if not already done. Safe to call multiple times, and only talks to the
   // authenticator if there is no PIN already. Defaults to 1234 if nothing else
   // is set. Fails if the PIN requirements are not satisfied.
-  Status SetPin(const cbor::Value::BinaryValue& new_pin_utf8 = {0x31, 0x32,
-                                                                0x33, 0x34});
+  Status SetPin(const cbor::Value::BinaryValue& new_pin_utf8);
+  // Sets the PIN as above, using a default.
+  Status SetPin();
```

Comment:
The reason to not use optional parameters is to avoid having `DefaultPin` a (I think private should work) method of `CommandState`?

Same for `AttemptGetAuthToken`.

Reply:
The style guide says:
> Default arguments are allowed on non-virtual functions when the default is guaranteed to always have the same value.

Since the new default changes based on the length, I decided to overload instead. Also, would the optional parameter even work? I thought it only takes static values or functions?

Reply:
Oh right, if the min PIN length can change during runtime, it's an issue.

I don't know how much we can put in a default argument, I never use them.